ipa_builder_elements/sles-extra-hardware/post-install.d/99-package-installs-extra-hardware

```diff
@@ -19,21 +19,6 @@ zypper install -y make automake libtool pkgconfig libaio-devel
 
 zypper install -y libconfig-devel libnl-devel readline-devel flex
 
-# lldpd
```

Comment:
Let's just comment this out and not remove it, it could be useful in the future. The "extra" hardware is an IPA concept and IPA can be used with LDAP .